### PR TITLE
Remove du

### PR DESCRIPTION
**Pull request by hagai245**

### CODE REVIEW

File: src/config/fieldNames.js [hunk 1]
```diff
@@ -165,6 +165,7 @@ module.exports = {
         dailyRun: 'daily_run',
         recoveryRun: 'recovery_run',
         immediateRun: 'immediate_run',
+        duRemove: 'du_remove',
```
Comment (Eli-BB8) [inline src/config/fieldNames.js:168]: remove this line

File: src/config/fieldNames.js [hunk 2]
```diff
@@ -173,6 +174,7 @@ module.exports = {
         add: 'add',
         update: 'update',
     },
+    lastDataFolderName: 'data/recovery_run',
```
Comment (Eli-BB8) [inline src/config/fieldNames.js:177]: unnecessary anymore

File: src/util/cleanDu/cleanDus.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const getIrrelevantDus = require('./getIrrelevantDus');
+const deleteDus = require('./deleteDus');
+const { logLevel } = require('../logger');
+const logDetails = require('../logDetails');
+const p =  require('../../config/paths');
+const AuthClass = require('../../auth/auth');
+
+async function cleanDu(dataSource, records, sendLog, recivedAuth) {
```
Comment (Eli-BB8) [inline src/util/cleanDu/cleanDus.js:8]: recivedAuth => Auth

File: src/runningMethods/daily.js [hunk 4]
```diff
@@ -40,4 +41,5 @@ const GetDataAndProcess = async (dataSource, akaData, sendLog, dataObj, func) =>
     // In case datasource is aka, I get data before function and therefore not need to get data again
     let data = dataSource === fn.dataSources.aka ? akaData : await func(dataSource, dataObj.data, dataObj.fileName);
     await diffsHandler(data, dataSource, akaData.all, fn.runnigTypes.dailyRun, sendLog);
+    await cleanDus(dataSource, data, sendLog);
```
Comment (Eli-BB8) [inline src/runningMethods/daily.js:44]: Auth?

File: src/util/cleanDu/cleanDus.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    const Auth = recivedAuth ? recivedAuth : new AuthClass(sendLog);
```
Comment (Eli-BB8) [inline src/util/cleanDu/cleanDus.js:9]: delete this line

File: src/util/cleanDu/cleanDus.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    const query = {
+        params: {
+            ['domainUsers.dataSource']: dataSource
```
Comment (Eli-BB8) [inline src/util/cleanDu/cleanDus.js:12]: ..... ,

File: src/util/cleanDu/cleanDus.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        }
+    }
+    const dataSourcePersons = await Auth.axiosKartoffel.get(
+        p().KARTOFFEL_PERSON_API, query).catch(err => {
```
Comment (Eli-BB8) [inline src/util/cleanDu/cleanDus.js:16]: no need to go down a line, after catch is enough

File: src/util/cleanDu/cleanDus.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        sendLog(logLevel.error, logDetails.error.ERR_GET_RAW_DATA , dataSource, err.message);
```
Comment (Eli-BB8) [inline src/util/cleanDu/cleanDus.js:17]: new log detail

File: src/util/cleanDu/deleteDus.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = async (domainUsers, dataSourcePersons, Auth, sendLog) =>{
+    if(domainUsers.length < dataSourcePersons.length / 2) {
+        for (const domainUser of domainUsers) {
+            let person = dataSourcePersons.find(personObj => personObj.domainUsers.find(domainUserObj => domainUserObj.uniqueID == domainUser))
+            let id = person._id;
+            try{
+                await Auth.axiosKartoffel.delete(p(id , domainUser).KARTOFFEL_DELETE_DOMAIN_USER_API);
+            } catch (err){
+                console.log(err)
```
Comment (Eli-BB8) [inline src/util/cleanDu/deleteDus.js:13]: logs

File: src/util/cleanDu/getIrrelevantDus.js [hunk 10]
```diff
@@ -0,0 +1,102 @@
+const assembleDomainUser = require('../fieldsUtils/assembleDomainUser');
+const fn = require('../../config/fieldNames');
+const diff = require('diff-arrays-of-objects');
+const fs = require("fs");
+const { filename } = require('winston-daily-rotate-file');
+const p = require('../../config/paths');
+
```
Comment (Eli-BB8) [inline src/util/cleanDu/getIrrelevantDus.js:7]: add require logDetails

**Review by NoamYes — APPROVED** [review]

**Pull request merged**